ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1348,7 +1348,7 @@ export const WorkspaceEdit = fp.flow(withCurrentWorkspace(), withCdrVersions(),
             <WorkspaceResearchSummary
                 researchPurpose={researchPurposeQuestions[2]}
                 researchValue={intendedStudy}
-                onChange={v => this.updateResearchPurpose('intendedStudy', v)}
+                onChange={v => this.updateResearchPurpose('intendedStudy', v.trim())}
```

Comment:
alt (or additional) fix: update validators to include a `presence` block.  This considers all-whitespace empty where `length.minimum` does not.

Reply:
Thanks, I'd forgotten about those validator details. I'm leaning to stick with this approach as I feel it may also be preferred to apply the trimmed value; thoughts on this?

Reply:
I agree that we want to store the trimmed value.

I'd prefer to also update the validators, to show intention.